src/staff_augmentation/components/HowItWorks/index.js

```diff
@@ -1,33 +1,65 @@
+import SVGIcon from 'main_app/components/SVGIcon'
+
+import classes from './styles.module.scss'
+
 function HowItWorks () {
   return (
-    <section>
-      <div>
-        <p>Why staff augmentation</p>
-        <h2>
-          <div>The extra talent needed to</div>
-          <div>Boost Your Projects</div>
-        </h2>
-      </div>
-      <div>
-        <figure>
-          <div>
-            
-          </div>
-          <figcaption>Dedicated Team Brief. Reduce liability and grow your business.</figcaption>
-        </figure>
-        <figure>
-          <div>
-            
-          </div>
-          <figcaption>Candidate Screening. Streamline your workflow and scale.</figcaption>
-        </figure>
-        <figure>
-          <div>
-            
-          </div>
-          <figcaption>Focus on Growing the Business. Streamline your workflow and scale.</figcaption>
-        </figure>
-      </div>
+    <section className={classes.howItWorksSection}>
+        <div className={classes.sectionHeader}>
+          <p className={classes.subheadingText}>How it works</p>
+          <h2 className={classes.titleText}>We make it <span>simple</span> for you</h2>
+        </div>
+        <ol className={classes.sectionContent}>
+          <li className={classes.transparentArrow}>
+            <SVGIcon name='staff_augmentation/transparent_arrow' />
+          </li>
+          <li className={classes.contentColumn}>
+            <div className={classes.arrowContainer}>
+              <div className={classes.arrowTip}></div>
+              <div className={classes.numericBadge}>
+                <p>1</p>
+              </div>
+            </div>
```

Comment:
- Add `aria-hidden="true"` to the `arrowTip` `div`. Also, this element can be closed without an extra tag: `<div />` instead of `<div></div>`
- The `p` inside the `numericBadge` `div` should not be necessary.
- You could extract this to a component to cleanup the HTML a bit. You could just place it in `./ArrowNumberBadge.js`. You'd then replace these 6 lines with `<ArrowNumberBadge>1</ArrowNumberBadge>`